README.md

```diff
@@ -1 +1,40 @@
-LearnAway is a tool to facilitate the self-learning created by two technology enthusiasts, who live far from each other but linked by the passion for programming.
\ No newline at end of file
+# learnaway
+
+LearnAway is a tool to facilitate the self-learning created by two technology enthusiasts, who live far from each other but linked by the passion for programming.
+
+## Requirements
+
+LearnAway api requires php and composer:
+
+### PHP
+
+```
+sudo apt update && sudo apt install wget php-cli php-zip unzip curl
+```
+### Composer
+```
+curl -sS https://getcomposer.org/installer |php
+sudo mv composer.phar /usr/local/bin/composer
+```
+
+### Installing api dependencies
+
+```
+cd api/
+composer install
```

Comment:
Running these from a vanilla Debian, I get the following error:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for symfony/framework-bundle v4.3.5 -> satisfiable by symfony/framework-bundle[v4.3.5].
    - symfony/framework-bundle v4.3.5 requires ext-xml * -> the requested PHP extension xml is missing from your system.
  Problem 2
    - symfony/framework-bundle v4.3.5 requires ext-xml * -> the requested PHP extension xml is missing from your system.
    - sensio/framework-extra-bundle v5.5.1 requires symfony/framework-bundle ^4.3|^5.0 -> satisfiable by symfony/framework-bundle[v4.3.5].
    - Installation request for sensio/framework-extra-bundle v5.5.1 -> satisfiable by sensio/framework-extra-bundle[v5.5.1].

  To enable extensions, verify that they are enabled in your .ini files:
    - /etc/php/7.3/cli/php.ini
    - /etc/php/7.3/cli/conf.d/10-opcache.ini
    - /etc/php/7.3/cli/conf.d/10-pdo.ini
    - /etc/php/7.3/cli/conf.d/20-calendar.ini
    - /etc/php/7.3/cli/conf.d/20-ctype.ini
    - /etc/php/7.3/cli/conf.d/20-exif.ini
    - /etc/php/7.3/cli/conf.d/20-fileinfo.ini
    - /etc/php/7.3/cli/conf.d/20-ftp.ini
    - /etc/php/7.3/cli/conf.d/20-gettext.ini
    - /etc/php/7.3/cli/conf.d/20-iconv.ini
    - /etc/php/7.3/cli/conf.d/20-json.ini
    - /etc/php/7.3/cli/conf.d/20-phar.ini
    - /etc/php/7.3/cli/conf.d/20-posix.ini
    - /etc/php/7.3/cli/conf.d/20-readline.ini
    - /etc/php/7.3/cli/conf.d/20-shmop.ini
    - /etc/php/7.3/cli/conf.d/20-sockets.ini
    - /etc/php/7.3/cli/conf.d/20-sysvmsg.ini
    - /etc/php/7.3/cli/conf.d/20-sysvsem.ini
    - /etc/php/7.3/cli/conf.d/20-sysvshm.ini
    - /etc/php/7.3/cli/conf.d/20-tokenizer.ini
    - /etc/php/7.3/cli/conf.d/20-zip.ini
  You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.
```

Do you have any clues? Should I use Ubuntu instead of Debian? Thanks!

Reply:
It didn't work on Ubuntu neither :disappointed:

Reply:
To work with Symfony framework you need to activate some PHP extension. I will edit the README to specify the necessary extensions.

Reply:
I have updated Pull Request with docker config to deploy a php container and also the README has updated steps to deploy api.

Regards!